test/test_memory_allocation_gpu.jl

```diff
@@ -191,8 +191,8 @@ end
     xyGrid = VecchiaMLE.generate_xyGrid(n)
     MatCov = VecchiaMLE.generate_MatCov(n, params, xyGrid)
     samples = VecchiaMLE.generate_Samples(MatCov, n, Number_of_Samples; mode=gpu)
-
-    model = VecchiaMLE.VecchiaModelGPU(samples, k, xyGrid)
+    input = VecchiaMLE.VecchiaMLEInput(n, k, samples, Number_of_Samples, 5, 1; ptGrid=xyGrid)
```

Comment:
Maybe rename it `iVecchiaMLE`to be consistent with `test/test_memory_allocation_cpu.jl`?